### PR TITLE
display author/committer mail in separate columns

### PR DESCRIPTION
**Pull request by antis81**
Just a small one. This makes it possible to hide the e-mail column - e.g. when doing screenshots :smile: :bulb:. Currently it is not possible to hide certain columns though - but here's one use case for this.

### CODE REVIEW

**Comment by antis81** [comment]
I bet we are ready to merge this.

File: History/HistoryModel.cpp [hunk 1]
```diff
@@ -171,8 +175,10 @@ QVariant HistoryModel::headerData( int section, Qt::Orientation orientation, int
     case colGraph:          return trUtf8( "Graph" );
     case colMessage:        return trUtf8( "Message" );
     case colAuthor:         return trUtf8( "Author" );
+    case colAuthorMail:     return trUtf8( "Author e-mail" );
```
Comment (scunz) [inline History/HistoryModel.cpp]: Should remove the dash here, probably. (same for committer below)

**Comment by scunz** [comment]
Other than that, this change is nice :)

File: History/HistoryModel.cpp (hunk 1 above)
Comment (antis81) [inline History/HistoryModel.cpp]: Hm, `e-mail` just looks nicer than `email` to me. Maybe we just should write `mail` then?

Comment (scunz) [inline History/HistoryModel.cpp]: Yeah, indeed just `mail` would probably fit even better.

**Pull request merged**